ansible/devutils

```diff
@@ -45,15 +47,18 @@ def get_conn_graph_facts(hosts):
     g_conn_graph_facts = conn_graph_helper.get_conn_graph_facts(hostnames)
     return g_conn_graph_facts
 
+
 def build_global_vars(concurrency, inventory):
     global g_task_runner, g_inv_mgr
     g_task_runner = TaskRunner(max_worker=concurrency)
     g_inv_mgr = HostManager(inventory)
 
+
```

Comment:
Why we add this blank line?

Reply:
In order to fix flake8 [E302](https://www.flake8rules.com/rules/E302.html) errors reported by pre-commit.